camelot/view/action_steps/select_file.py

```diff
@@ -86,7 +86,7 @@ def gui_run(self, gui_context):
                                     caption=six.text_type(self.caption),
                                     directory=directory,
                                     filter=self.file_name_filter)
-            if qt_api == 'PyQt5':
+            if qt_api in ['PyQt5', 'PyQt6']:
```

Comment:
ter info : gezien we nu netjes met verschillende branches voor vanalles en nog wat, zijn die 'if' constructs eigenlijk niet meer nodig, en kunnen we er rustig vanuit gaan dat het PyQt6 is